package.json

```diff
@@ -29,6 +31,11 @@
     "rxjs": "6.4.0"
   },
   "devDependencies": {
+    "@bazel/bazel": "latest",
+    "@bazel/buildifier": "latest",
```

Comment:
`buildifier` is never used to format or lint bazel files anywhere in the `scripts` section, so it could be safely removed here.
Or you can add formatting like this (this will **not** work on Windows though):
```
  "scripts": {
    "bazel:format": "find . -type f \\( -name \"*.bzl\" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs buildifier -v --warnings=attr-cfg,attr-license,attr-non-empty,attr-output-default,attr-single-file,constant-glob,ctx-actions,ctx-args,depset-iteration,depset-union,dict-concatenation,duplicated-name,filetype,git-repository,http-archive,integer-division,load,load-on-top,native-build,native-package,out-of-order-load,output-group,package-name,package-on-top,positional-args,redefined-variable,repository-name,same-origin-load,string-iteration,unsorted-dict-items,unused-variable",
    "bazel:lint": "yarn bazel:format --lint=warn",
    "bazel:lint-fix": "yarn bazel:format --lint=fix",
    // ...
  }
```